src/test/kotlin/com/hypto/iam/server/apis/CredentialApiKtTest.kt

```diff
@@ -477,4 +478,42 @@ internal class CredentialApiKtTest : AbstractContainerBaseTest() {
             }
         }
     }
+
+    @Nested
+    @DisplayName("List credentials API tests")
+    inner class ListCredentialTest {
+        @Test
+        fun `list credentials of a user`() {
```

Comment:
Add cases for
1. User or organization missing should throw 400
2. Zero credentials found for user should return empty array
3. List should not contain the secret.

Reply:
Done